docs/1.0/api/size.md

```diff
@@ -34,6 +34,7 @@ Sets how the image is fitted to its target dimensions.
 - `contain`: Default. Resizes the image to fit within the width and height boundaries without cropping, distorting or altering the aspect ratio.
 - `max`: Resizes the image to fit within the width and height boundaries without cropping, distorting or altering the aspect ratio, and will also not increase the size of the image if it is smaller than the output size. 
 - `fill`: Resizes the image to fit within the width and height boundaries without cropping or distorting the image, and the remaining space is filled with the background color. The resulting image will match the constraining dimensions.
+- `fill-max`: Resizes the image to fit within the width and height boundaries with cropping but without distorting the image. The resulting image will match the constraining dimensions.
```

Comment:
This page is for v1. The feature is not going to be backported.

Reply:
You are right, I have removed it